person.rb

```diff
@@ -0,0 +1,17 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(age, name: 'unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def private_of_age?
+    @age >= 18
+  end
+
+  def public_can_use_services?
+    private_of_age? || @parent_permission
+  end
```

Comment:
- [ ] As required for the project, kindly ensure that the `#of_age?` method is set as `Private` method as right now all methods automatically `Public`. You can update them to look similar to this example below(or use another way to make it private):
```ruby
class Person
.
.
.
def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end

  def can_use_services? 
    of_age?() || @parent_permission
  end

  private 

  def of_age?
    @age >= 18
  end
end
```